lib/msal-browser/src/telemetry/BrowserPerformanceClient.ts

```diff
@@ -128,6 +130,48 @@ export class BrowserPerformanceClient
         return base64Encode(JSON.stringify(thumbprint));
     }
 
+    /**
+     * Generates serialized thumbprint map
+     * @return string
+     */
+    getThumbprintMapSerialized(): string {
+        const toArray = (map : any) : any =>
```

Comment:
You can probably simplify this with `const stringifiedThumbprints = JSON.stringify(Array.from(this.thumbprints.entries()))`